packages/utils-connector-tools/tests/requester.ts

```diff
@@ -206,9 +204,9 @@ const loopServerMultiSteps = {
 };
 
 test(`Requester follows all hops, reports the right number and returns the final string content`, async (t) => {
-    const { requester, server } = t.context;
+    const { requester } = t.context;
 
-    server.configure(hopsServerConfig);
+    const server = await Server.create({ configuration: hopsServerConfig });
```

Comment:
Does this need a corresponding `await server.stop()`? I'm only marking this one, but there are multiple occurances in the file.

Reply:
yes, that's why it was failing on Linux and mac at the moment. I've pushed the changes